### ISSUE TITLE
about crop video

### ISSUE THREAD

**Issue by newExplore-hash** [AliaksandrSiarohin/first-order-model#404]
hi AliaksandrSiarohin,
so awesome project, congradulations for you. about cropping video, I found that it is a cropping method for offline videos(crop box computed by all frames), If I want to crop the real-time video frame in the camera，Is there a more reasonable way to crop?

**Comment by rizwanishaq** [comment]
@newExplore-hash did you able to get cropping in real-time?

**Comment by newExplore-hash** [comment]
yeah, I want to drive a picture in real time through the camera，but I can't seem to find a suitable crop method.

**Comment by rizwanishaq** [comment]
@newExplore-hash  [this link](https://stackoverflow.com/questions/44650888/resize-an-image-without-distortion-opencv ) can help.

**Comment by oyatmicah** [comment]
actually in the old demo model, somebody was able to solve this issue but I am also finding it difficult to crop the video too, and I believe it should be possible to do this automatically at upload time
